### PR TITLE
Initial changeset for adding test for bigtable 

### PR DESCRIPTION
**Pull request by ricky3dec**
Review only PR.

### CODE REVIEW

**Comment by ricky3dec** [comment]
Hello @yongtang, please help with following queries:

1. In "bigtable_dataset_ops.py" Whether to use the core_ops “io_big_table_client” or use the python bigtable_api’s BigtableClient?
2. The emulator does set an enviornment variable “BIGTABLE_EMULATOR_HOST”, but would tensorflow’s bigtable client be able to make connection on this endpoint? I assume there is no endpoint in bigtable’s client.

Thanks

File: tensorflow_io/core/python/experimental/bigtable_dataset_ops.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import tensorflow as tf
+from tensorflow_io.core.python.ops import core_ops
+from tensorflow_io.bigtable.python.ops import bigtable_api;
+
+class BigTableIODataset(tf.data.Dataset):
+    """BigTableIODataset"""
+
+    def __init__(self, project=None, instance=None, table=None):
+        """BigTableIODataset."""
+        return;
```
Comment (ricky3dec) [inline tensorflow_io/core/python/experimental/bigtable_dataset_ops.py:27]: Hello @yongtang, I am bit confused:

1. Whether to use the core_ops “io_big_table_client” here or use the python bigtable_api’s BigtableClient?
2. The emulator does set an enviornment variable “BIGTABLE_EMULATOR_HOST”, but would tensorflow’s bigtable client be able to make connection? I assume there is no endpoint in bigtable’s client.

Thanks

**Comment by vlasenkoalexey** [comment]
FYI, here is a test for BigTable ops before in TensorFlow contrib before it was deleted: https://github.com/tensorflow/tensorflow/blob/v1.15.2/tensorflow/contrib/bigtable/python/kernel_tests/bigtable_ops_test.py

You might find it useful.

**Comment by ricky3dec** [comment]
Thanks @vlasenkoalexey for your help! Will look through.

**Pull request closed without merging**